packages/mobile/src/app/saga.ts

```diff
@@ -310,10 +327,58 @@ export function* handleSetAppState(action: SetAppState) {
   }
 }
 
+export function* handleSaveOtaTranslations(language: string, translations: Resource) {
+  const hasPreviouslyFetchedTranslations = yield RNFS.exists(OTA_TRANSLATIONS_FILEPATH)
+  if (hasPreviouslyFetchedTranslations) {
```

Comment:
I just wanted to double-check: is this existence check necessary?  It might cause a time-to-check-time-to-use bug with reading.  Consider this inter-leaving:

1. https://github.com/valora-inc/wallet/pull/1433/files#diff-42a622ab015727ac614a8c7c378e96f0ddd2a323930c2c3541c29c8fdeba37a7R23
2. https://github.com/valora-inc/wallet/pull/1433/files#diff-ff3f8a62a1a862b5c63324a903a40de66fef2fe88527f5024500dfb574a19758R333
3. https://github.com/valora-inc/wallet/pull/1433/files#diff-42a622ab015727ac614a8c7c378e96f0ddd2a323930c2c3541c29c8fdeba37a7R25

The atomic way to update contents is with:
1. create new tmp file with new contents
2. move tmp file to OTA_TRANSLATIONS_FILEPATH location

Reply:
good one! i can't remember why i added this actually, i was under the impression that `writeFile` appends instead of overwrites the file contents but this is wrong. i just tested and it looks like `writeFile` does the overwriting already, so i removed this check :)